build/Targets/VSL.Versions.targets

```diff
@@ -73,8 +73,8 @@
 
   <!-- NuGet version -->
   <PropertyGroup>
-    <CommonNuGetMoniker>beta2</CommonNuGetMoniker>
-    <VSSemanticVersion>15.3</VSSemanticVersion>    
+    <CommonNuGetMoniker>beta3</CommonNuGetMoniker>
```

Comment:
@basoundr should confirm but I think that's set only for local builds and somewhere later it gets overridden to set 15.6.0-beta1. And yes we should document this\add comments.